.github/workflows/scripts/win_build.ps1

```diff
@@ -69,7 +69,8 @@ if ($llvmVer -eq "10") {
     throw "Unsupported LLVM version"
 }
 
-$env:TAICHI_CMAKE_ARGS += " -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_C_COMPILER=clang"
+$env:TAICHI_USE_MSBUILD = 1
```

Comment:
Where is it used? I can only see assignments but not reads.

Reply:
It's a flag for `setup.py`